### DISCUSSION TITLE
ROI_6872

### DISCUSSION THREAD

**Discussion by csaybar** [cloudsen12/models#233] [Lissette]
**20190607T171909_20190607T173611_T13QGD**
![image](https://user-images.githubusercontent.com/16768318/156869105-a0d4afd9-a2f2-428d-83bc-36a9106c62d4.png)
![image](https://user-images.githubusercontent.com/16768318/156869123-b768bb6e-fa72-4b33-a616-0db65c277643.png)

**20190915T171929_20190915T173255_T13QGD**
![image](https://user-images.githubusercontent.com/16768318/156869142-4f20af4d-c66f-4fd4-91ac-40cf787c749e.png)

**20191030T172441_20191030T173610_T13QGD**
![image](https://user-images.githubusercontent.com/16768318/156869173-224e7ef3-33fc-48ca-a1fc-577331fc9fec.png)

**Comment by ThomasGorisse** [comment]
SceneView v4.0.0-rc.1 is available now — a complete rewrite with Jetpack Compose.

Latest:
- `io.github.sceneview:sceneview:4.0.0-rc.1`
- `io.github.sceneview:arsceneview:4.0.0-rc.1`

Release notes: [v4.0.0-rc.1](https://github.com/sceneview/sceneview/releases/tag/v4.0.0-rc.1)

Stable v4.0.0 coming soon. The MCP server is already on npm `@latest`: `npx sceneview-mcp`.
